test/unit/sql/where.test.ts

```diff
@@ -2246,6 +2246,39 @@ describe(getTestDialectTeaser('SQL'), () => {
 
     if (sequelize.dialect.supports.JSONB) {
       describe('JSONB', () => {
+        if (dialectSupportsArray()) {
```

Comment:
this check is not necessary. Only postgres supports JSONB & it supports arrays.
If a dialect starts supporting these operators but they work with something else than arrays, we'll adapt the test suite

Reply:
Thanks